client/actions/graphql/eventGQLActions.js

```diff
@@ -75,7 +75,14 @@ export const createEvent = ({
 ).then(data => dispatch({
   type: CREATE_EVENT, payload: data.data, error: false,
 }))
-  .catch(error => handleError(error, dispatch));
+  .catch((error) => {
+    if (error.toString().includes('Calendar API not authorized')) {
+      const authUrl = error.graphQLErrors[0].AuthUrl;
+      window.location.href = authUrl;
```

Comment:
I am not sure about this. We can try use` browserHistory.push` for `history.push` for redirecting since we are suing react-router. But lmk if this is not possible.